tests/personal_data/portfolios/test_portfolio.py

```diff
@@ -64,18 +64,34 @@ async def test_copy_portfolio(backtesting_trader):
 
 
 async def test_get_portfolio_from_amount_dict(backtesting_trader):
+    # todo fix
```

Comment:
Is it fixed ?

Reply:
woops, yes it is

Reply:
remove here https://github.com/Drakkar-Software/OctoBot-Trading/pull/938